code/modules/projectiles/projectile/magic.dm

```diff
@@ -131,18 +131,13 @@
 	switch(randomize)
 		if("monkey")
 			new_mob = new /mob/living/carbon/monkey(M.loc)
+
 		if("robot")
-			var/robot = pick("random_cyborg","syndiborg","drone")
-			var/path
-			switch(robot)
-				if("random_cyborg")
-					path = pick(typesof(/mob/living/silicon/robot/modules) - typesof(/mob/living/silicon/robot/modules/syndicate))
-					new_mob = new path(M.loc)
-				if("syndiborg")
-					path = pick(typesof(/mob/living/silicon/robot/modules/syndicate))
-					new_mob = new path(M.loc)
-				if("drone")
-					new_mob = new /mob/living/simple_animal/drone/polymorphed(M.loc)
+			var/robot = pick(200;/mob/living/silicon/robot,
+							/mob/living/silicon/robot/modules/syndicate,
+							/mob/living/silicon/robot/modules/syndicate/medical,
+							200;/mob/living/simple_animal/drone/polymorphed)				
```

Comment:
Um...

Reply:
pickweight is default 100 so 200 is twice as much.

It's equivalent to as it was previously.

Reply:
http://www.byond.com/docs/guide/chap16.html#16.1.4